media-libs/freetype/freetype-2.11.0.recipe

```diff
@@ -16,7 +16,7 @@ PATCHES="
 #PATCHES="freetype-subpixel.patchset" # enable patented subpixel rendering
 
 ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Not needed atm, there is no x86_gcc2 for secondary architecture (yet), but there are other recipes still mentioning it)